### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Adam777Z** [Adam777Z/xbox-controller-button-remapper#2] [Help]
If it is not working for you, please read the [ReadMe](https://github.com/Adam777Z/xbox-controller-button-remapper#readme) and make sure you are not missing anything.
- Check that you are using the latest version and no program files are missing. Download and extract again to make sure.
- The controller is connected and working
- You are running Windows 10/11 64-bit and it is up to date
- [Microsoft Visual C++ Redistributable for Visual Studio 2019 x64](https://visualstudio.microsoft.com/downloads/#microsoft-visual-c-redistributable-for-visual-studio-2019) is installed
- The controller is updated to the latest firmware version. (Instructions: [Update your Xbox Wireless Controller](https://support.xbox.com/en-US/help/hardware-network/controller/update-xbox-wireless-controller))
- Make sure that you do not have too many controllers connected. Disconnect the other controller(s) and try again.
- Check the configuration (config.ini file)
- Check that nothing else is causing it (controller configured in other program, etc.)

If it is still not working for you, comment with the following:
- Xbox Controller button remapper program version
- Operating system (OS) details (name, edition, version, build) that you are using
- Xbox Controller model (check it on [Wikipedia](https://en.wikipedia.org/wiki/Xbox_Wireless_Controller))
- Controller connection (wired or wireless, if wireless: Xbox Wireless or Bluetooth, if Xbox Wireless: Xbox Wireless Adapter model)
- The name of the game/program it is not working in
- Does it work in any other game/program?
- What button(s) and key(s) you configured (only if you changed the defaults)
- Does it work with another button and/or key?

**Comment by KreeperSteve** [comment]
I'm running Windows 10 21H1 19043.1266
I am using the Xbox series controller wireless via Xbox wireless dongle. It seems to work in Chrome but not Skyrim special edition I have the share button mapped to Windows Alt G as a single press.

**Reply by Adam777Z** [reply]
Did you configure the controller in Steam?
Make sure it's set to the default settings.

**Reply by KreeperSteve** [reply]
Yes I'm using the default steam controller settings and it's still not working let me try another game it works in Chrome & window settings

**Comment by thinh105** [comment]
Hi,

I have to say, your work is awesome, it brings a lot of meaning to my gaming life

I just have a small problem, hope you can help me,

Everything is working fine when I turn on my controller before running Xbox-controller-button-remapper,

When my controller is off while Xbox-controller-button-remapper is running.

then I turn my controller on, the light is blinking and then off in 3s

I only can turn my controller on if I turn off xbox-controller-button-remapper in the system tray,

in config file, I just change one line
`share_key = 29,67`

(ctrl + F9)

Maybe I miss something, can you help me, thanks


**update:**

in config file, I have to comment out these lines, and everything is back to normal, I can turn on my controller normally,

```
;xbox_hold_mode = 1
;xbox_longpress_key = 1
;xbox_longpress_duration = 1000
;xbox_delay = 0
;xbox_duration = 1
```

**Reply by Adam777Z** [reply]
That's interesting.

It's good that you found the fix for the issue.

**Reply by thinh105** [reply]
@Adam777Z  it still comes back, do you also have that error?

When I let my controller alone, it auto turn off, and then I want to turn it on, cannot do that, it auto-off after 2-3s

all I can do is turn off your program, and everything back to normal

**Reply by Adam777Z** [reply]
What operating system (OS) are you using?
What controller(s) are you using?

**Reply by thinh105** [reply]
> What operating system (OS) are you using? What controller(s) are you using?

Windows 10 21H1

Xbox one series S ( I have just 1 controller )

here is my config
[config.ini.txt](https://github.com/Adam777Z/xbox-controller-button-remapper/files/7507352/config.ini.txt)

**Reply by Adam777Z** [reply]
Set the Xbox button to 0 (Disabled, no remapping is done) in the config.ini file and try again.
`xbox_key = 0`

**Reply by thinh105** [reply]
still not working,

here is the video: https://www.youtube.com/watch?v=801L7fZQ6_8

if I restart xbox-controller-button-remapper, and try to turn on the controller, it's working well,

Somehow, it only work in the first turn on,

**Reply by Adam777Z** [reply]
Is the controller updated to the latest firmware version?

**Reply by thinh105** [reply]
> Is the controller updated to the latest firmware version?

Yes, I think so,

**Reply by Adam777Z** [reply]
Please check and make sure that the controller is updated to the latest firmware version.

**Reply by thinh105** [reply]
> Please check and make sure that the controller is updated to the latest firmware version.

Yes, because I run the Xbox app to update every day


is there any way I can send you the log? it's kind of hard to debug like this :(

**Reply by Adam777Z** [reply]
Instructions: [Update your Xbox Wireless Controller](https://support.xbox.com/en-US/help/hardware-network/controller/update-xbox-wireless-controller)

**Reply by thinh105** [reply]
![image](https://user-images.githubusercontent.com/23691294/142713076-9c0ecc20-dfd5-4349-aea3-dbc156d1f2b1.png)
Yes, I updated it every time since I bought it

**Reply by Adam777Z** [reply]
There's a new version available now. Please try it.

**Reply by thinh105** [reply]
```
INFO: Debug mode enabled.
INFO: Event: 1541
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1617
INFO: Event: 1539
INFO: Xbox button pressed.
INFO: Event: 1618
INFO: Event: 1540
INFO: Event: 1616
INFO: Event: 1616
INFO: Event: 1616
INFO: Event: 1616
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Xbox button released.
INFO: Opened Xbox Game Bar.
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
```

here is my log,

after the 1st success, when I turn it off and on, it automatically turns off

( in the second try, it show the Xbox Game bar and off. After that, when try to turn on, Xbox Game bar did not show anymore )

**Reply by Adam777Z** [reply]
> Set the Xbox button to 0 (Disabled, no remapping is done) in the config.ini file and try again.
> `xbox_key = 0`

**Reply by thinh105** [reply]
Still happends,

```
; Only works with this setting disabled:
; Settings - Gaming - Xbox Game Bar - Open Xbox Game Bar using this button on a controller: (X)

[settings]
; Enable debug mode to show debug messages in a console window
; Enable: 1
; Disable: 0 (default)
debug = 1

[controller1]
; Map the Share button on the controller

; Decimal key code, see the bottom of this file for a list
; Multiple keys are supported. Separate multiple keys with commas.
; If you set it to 0, set longpress_key correctly, and set hold_mode = 2,
; tapping the button for less than longpress_duration will do nothing.
; Default: 91,56,84 (Xbox Game Bar - Take screenshot: WINDOWS+ALT+PRINT SCREEN)
share_key = 91,56,84

; If 1, "key" will be sent for as long as button is pressed and longpress_*
; will be ignored - simple as that.
; If 2, "key" will be briefly tapped if button is held for less than
; longpress_duration, and longpress_key will be briefly tapped if the button
; is held for longer than that.
; Note that even after the button has been held for longer than longpress_duration,
; you still need to release it before anything will be sent.
; Default: 2
share_hold_mode = 2

; See above.
; Multiple keys are supported. Separate multiple keys with commas.
; Default: 91,56,19 (Xbox Game Bar - Start/stop recording: WINDOWS+ALT+R)
share_longpress_key = 91,34

; See above. In milliseconds (1000 = 1 second)
; Default: 1000
share_longpress_duration = 1000

; Wait this long before tapping a key (whether longpress or not).
; In milliseconds
; Ignored if hold_mode = 1
; In hold_mode = 2, wait this many ms after the button is released before sending
; the keypress (whether key or longpress_key).
; Don't make it too long as other presses will not be detected during this wait.
; Default: 0
share_delay = 0

; Wait this long before releasing a key (whether longpress or not).
; In milliseconds
; Ignored if hold_mode = 1
; In hold_mode = 2, wait this many ms before releasing the key (whether key or longpress_key).
; Don't make it too long as other presses will not be detected during this wait.
; Default: 1
share_duration = 1

; Map the Xbox/Guide button on the controller

; See above.

; Default: 91,34 (Open Xbox Game Bar: WINDOWS+G)
xbox_key = 0

; Default: 1
xbox_hold_mode = 1

; Default: 1
xbox_longpress_key = 1

xbox_longpress_duration = 1000
xbox_delay = 0
xbox_duration = 1

; End of controller1.
; Settings for other controllers follow
```

log here:  
```
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Xbox button released.
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
INFO: Controller 1 added.
INFO: Mapping: 030000005e040000120b000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: Opened controller 1: Xbox Controller
INFO: Event: 1541
INFO: Controller 1 removed.
INFO: Closed controller 1: Xbox Controller
INFO: Event: 1542
```

**Reply by fata1-err0r** [reply]
i have same problem too

**Reply by Adam777Z** [reply]
Are you connecting the controller via Bluetooth?

Did you try it with the Xbox Wireless Adapter?

**Reply by fata1-err0r** [reply]
i checked. the problem happens only with the Xbox Wireless Adapter.
Via Bluetooth works  fine.

**Reply by thinh105** [reply]
> i checked. the problem happens only with the Xbox Wireless Adapter. Via Bluetooth works fine.

Correct, I'm using the Xbox Wireless Adapter, maybe that's why the error happened

**Reply by Adam777Z** [reply]
There's a new version available now. Please try it.

**Reply by fata1-err0r** [reply]
> There's a new version available now. Please try it.

still doesnt work correctly with adapter

**Reply by robbely** [reply]
There's a whole bunch of people with this problem in the "issues" section. Is this still being looked into?

**Reply by Adam777Z** [reply]
> There's a whole bunch of people with this problem in the "issues" section. Is this still being looked into?

Yes, please see the issue for updates: #6

**Reply by Adam777Z** [reply]
This issue is fixed in version [1.5.0.0](https://github.com/Adam777Z/xbox-controller-button-remapper/releases/tag/1.5.0.0).

**Comment by lighticore** [comment]
I downloaded this just to remap Share button to my "X" button on keyboard . I'm not sure how to do it, everytime i change something its just doesnt work.

**Reply by lighticore** [reply]
Seems like i managed to change the share button. But it doesnt work for in games. Is there a way to fix it?

**Reply by Adam777Z** [reply]
Please read the first post and provide more information.

**Comment by Shituation** [comment]
Not working here. 
Wanting to map guide button to F10.
![1](https://user-images.githubusercontent.com/43975882/206612528-3de0f732-af50-494a-9e70-7b5eee89be25.jpg)


```
; Only works with this setting disabled:
; Settings - Gaming - Xbox Game Bar - Open Xbox Game Bar using this button on a controller: (X)

[settings]
; Enable debug mode to show debug messages in a console window
; Enable: 1
; Disable: 0 (default)
debug = 0

[controller1]
; Map the Share button on the controller

; Decimal key code, see the bottom of this file for a list
; Multiple keys are supported. Separate multiple keys with commas.
; If you set it to 0, set longpress_key correctly, and set hold_mode = 2,
; tapping the button for less than longpress_duration will do nothing.
; Default: 91,56,84 (Xbox Game Bar - Take screenshot: WINDOWS+ALT+PRINT SCREEN)
share_key = 91,56,84

; If 1, "key" will be sent for as long as button is pressed and longpress_*
; will be ignored - simple as that.
; If 2, "key" will be briefly tapped if button is held for less than
; longpress_duration, and longpress_key will be briefly tapped if the button
; is held for longer than that.
; Note that even after the button has been held for longer than longpress_duration,
; you still need to release it before anything will be sent.
; Default: 2
share_hold_mode = 2

; See above.
; Multiple keys are supported. Separate multiple keys with commas.
; Default: 91,56,19 (Xbox Game Bar - Start/stop recording: WINDOWS+ALT+R)
share_longpress_key = 68

; See above. In milliseconds (1000 = 1 second)
; Default: 1000
share_longpress_duration = 1000

; Wait this long before tapping a key (whether longpress or not).
; In milliseconds
; Ignored if hold_mode = 1
; In hold_mode = 2, wait this many ms after the button is released before sending
; the keypress (whether key or longpress_key).
; Don't make it too long as other presses will not be detected during this wait.
; Default: 0
share_delay = 0

; Wait this long before releasing a key (whether longpress or not).
; In milliseconds
; Ignored if hold_mode = 1
; In hold_mode = 2, wait this many ms before releasing the key (whether key or longpress_key).
; Don't make it too long as other presses will not be detected during this wait.
; Default: 1
share_duration = 1

; Map the Xbox/Guide button on the controller

; See above.

; Default: 91,34 (Open Xbox Game Bar: WINDOWS+G)
xbox_key = 68

; Default: 1
xbox_hold_mode = 1

; Default: 1
xbox_longpress_key = 1

xbox_longpress_duration = 1000
xbox_delay = 0
xbox_duration = 1

; End of controller1.
; Settings for other controllers follow

[controller2]
share_key = 68
share_hold_mode = 2
share_longpress_key = 91,56,19
share_longpress_duration = 1000
share_delay = 0
share_duration = 1
xbox_key = 68
xbox_hold_mode = 1
xbox_longpress_key = 1
xbox_longpress_duration = 1000
xbox_delay = 0
xbox_duration = 1

[controller3]
share_key = 91,56,84
share_hold_mode = 2
share_longpress_key = 91,56,19
share_longpress_duration = 1000
share_delay = 0
share_duration = 1
xbox_key = 91,34
xbox_hold_mode = 1
xbox_longpress_key = 1
xbox_longpress_duration = 1000
xbox_delay = 0
xbox_duration = 1

[controller4]
share_key = 91,56,84
share_hold_mode = 2
share_longpress_key = 91,56,19
share_longpress_duration = 1000
share_delay = 0
share_duration = 1
xbox_key = 91,34
xbox_hold_mode = 1
xbox_longpress_key = 1
xbox_longpress_duration = 1000
xbox_delay = 0
xbox_duration = 1

; More controllers are supported. To configure them, add them like above: controller5, controller6, and so on.
; Otherwise, defaults are used.

```

**Reply by Shituation** [reply]
Ridiculously I found a post of someone from 7 years ago on reddit saying to enable game bar (if disabled like me) enable the button to open it, enter a custom shortcut key for the default "windows + G", then disable everything back again.

```
User avatar
level 1
javisMG
·
7 yr. ago
i7 6700k@4,7Ghz - nVidia GTX 780 @ 1200/1700

open xbox app, enable game dvr if disabled. press windows+g. configure guide button. disable game dvr. done
```

**Reply by Adam777Z** [reply]
Link to comment: https://www.reddit.com/r/pcgaming/comments/3udzes/comment/cxfr4sd/

Did you try that? Did it work?

**Reply by Shituation** [reply]
Yes, it was a problem on my system (didn't recognize the button when pressed) not the remapper. Apparently when re-configuring the button from windows settings makes the settings be valid again (even if you turn the game bar off) and windows starts paying attention to the button again. 🤷‍♂️
In my case, this was all about yuzu emu using the guide button for some commands, after doing this, the button gets recognized without the remapper.

**Reply by Adam777Z** [reply]
OK, that is good to know.

**Comment by taranchic** [comment]
Can't make the app to start up automatically. Added it to startup folder and it is also active in Task manager startup, but still it wont start with PC

**Reply by Shituation** [reply]
You dont have to put the exe in the startup folder, you have to put a shorcut to it there.

**Comment by taranchic** [comment]
Yes, of course I meant shortcut. Now I also tried to add it though
registry, and still it won't start up.

Shituation ***@***.***> schrieb am Mi., 1. Feb. 2023, 22:50:

**Reply by Shituation** [reply]
just to be sure;
WIN+R
'shell:startup' + enter
put the shorcut to the app there

(make sure app runs properly manually too)

**Comment by taranchic** [comment]
Somehow it worked after putting the shortcut into folder for the third
time. Though it seems to me it worked because I turned pc off and back on
instead of restarting. (Fast startup is turned off, so I don't know what
difference it could make)

Shituation ***@***.***> schrieb am Mi., 1. Feb. 2023, 22:59:

**Comment by XJIOP** [comment]
Not working on Xbox One Wireless Controller, but working with this app https://github.com/pinumbernumber/Xbox-360-Guide-Button-Remapper

**Reply by Shituation** [reply]
I have an Xbox one controller, and it works here... 
Read above, there is some bug on windows, try that.

**Reply by XJIOP** [reply]
I've tried everything I've read above and nothing helps.

In debugging only shows 1616 and 1536 events when the xbox button is pressed.
```
INFO: 2023-02-24 17:39:47: Debug mode enabled.
INFO: 2023-02-24 17:39:54: Controller 1 added.
INFO: 2023-02-24 17:39:54: Mapping: 030000005e040000e002000000007200,Xbox Controller,platform:Windows,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,misc1:b11,
INFO: 2023-02-24 17:39:54: Opened controller 1: Xbox Controller
INFO: 2023-02-24 17:39:54: Event: 1541
INFO: 2023-02-24 17:40:05: Event: 1616
INFO: 2023-02-24 17:40:05: Event: 1536
INFO: 2023-02-24 17:40:05: Event: 1616
INFO: 2023-02-24 17:40:05: Event: 1536
```

It works in Xbox-360-Guide-Button-Remapper, but there is no key combination.

**Reply by XJIOP** [reply]
After updating the firmware, it worked! Last time I updated the firmware a year ago and thought there would be no more updates because the device is not new.

Thanks for this app!

**Comment by AffiniaAurora** [comment]
It is not working for me when running Genshin Impact.

Program version :1.6.0
OS: Windows 10 22h2
Xbox Controller model : Xbox controller 2020
Controller connection : wired
The name of the game : Genshin Impact
Does it work in any other game/program : Yes
Does it work with another button and/or key? : No.

**Comment by NuggetDerj** [comment]
Program works flawlessly for me, as I was able to configure everything without much issue. However, it seems as though the remapper also gets triggered by Nintendo Switch controllers on inputs that are commonly used (R button registers as the guide button, D-Pad Up registers as the share button.) The one I'm using in particular is an 8BitDo Pro+ set to Switch mode, NOT Xinput mode. I find this very odd and inconvenient, as I often use one or the other for certain games. Is there any way of fixing this? Having the program differentiate between other kinds of controller API from Xinput? This would help a TON, and potentially more users in the future.

**Comment by KevTheGalaxybender** [comment]
I'm trying to map a key to a Xbox Series X controller and I can't get it to work. If i wanted to map the button to the keystroke 7, how do I do it? I tried using option 71 in the .ini but no luck.

**Comment by DOC2016** [comment]
Hi, I found this program while researching for a way to remap the share button to the media play/pause key. I like to listen to music while playing single player games, but during dialogs i need to pause it and i'm to lazy to reach the key on my keyboard.
I did a bit of research before writing this comment and from my understanding the media keys don't have scan codes on QBasic (I'm definitely not a programmar so my understanding on the topic is very limited and could be wrong, sorry for that). So I just want to know if I'm correct and currently it's not possible. And if so, if it's possible to implement it in the future.
Thank you in advance.

**Reply by Adam777Z** [reply]
It is possible, but it requires an update to the program.

**Reply by DOC2016** [reply]
Thank you for the quick response. I'll keep an eye on that to see if you manage to do it. Have a great day.

**Reply by Adam777Z** [reply]
Added in version [2.0.0.0](https://github.com/Adam777Z/xbox-controller-button-remapper/releases/tag/2.0.0.0).

**Reply by DOC2016** [reply]
Works flawlessly. Thanks a lot.
